### PR TITLE
Task#35: Adding classes ConnectionInfo, InvokeMainTestCase and Connec…

### PR DESCRIPTION
**Pull request by dp4295**
Let me know if you any question/feedback

### CODE REVIEW

File: src/main/java/edu/pdx/cs/InvokeMainTestCase.java [hunk 1]
```diff
@@ -0,0 +1,135 @@
+package edu.pdx.cs;
```
Comment (Evan-Kent) [inline src/main/java/edu/pdx/cs/InvokeMainTestCase.java:1]: This code isn't used in this PR and is part of another body of work. Can you remove this file and push another commit to your branch with the change? Please also make sure to cite your usage whenever you're using code from another location. If you don't, you are plagiarizing, which can place us all at risk.

File: src/test/java/edu/pdx/cs/ConnectionInfoTest.java [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+import java.util.regex.Pattern;
+import static org.hamcrest.MatcherAssert.assertThat;
+
+
+public class ConnectionInfoTest extends InvokeMainTestCase{
+
+
+    private InvokeMainTestCase. MainMethodResult invokeMain(String... args) {
+        return invokeMain(App.class, args);
+    };
```
Comment (Evan-Kent) [inline src/test/java/edu/pdx/cs/ConnectionInfoTest.java:17]: Remove this code and the `extends InvokeMainTestCase` part of class declaration.

An atomic test case for a getter might look something like the following (I'm bad at test names)
```
@Test
public void getPortReturnsExpected() { 
ConnectionInfo c = new ConnectionInfo("host", 22, "aang");
assertThat(c.getPort(), equalTo(22));
}
```

File: src/main/java/edu/pdx/cs/ConnectionInfo.java [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+
+/**
+ * This class is represents a <code>ConnectionInfo</code>
+ * This class has necessary field to create an FTP connection
+ * using getter methods.
+ */
+public class ConnectionInfo {
+
+    private Path currentDir;
+    private String IP;
```
Comment (Evan-Kent) [inline src/main/java/edu/pdx/cs/ConnectionInfo.java:13]: Current directory is a good fit for another part of the program. We can probably hold off on implementing it for now.

Can you rename `IP` to something like `host`? To reflect that we may be using IP or URL.

port can be an int. Check out the signature of this method in SocketClient.
`public void connect(String hostname, int port)`
https://commons.apache.org/proper/commons-net/apidocs/org/apache/commons/net/SocketClient.html

File: src/main/java/edu/pdx/cs/InvokeMainTestCase.java (hunk 1 above)
Comment (dp4295) [reply src/main/java/edu/pdx/cs/InvokeMainTestCase.java:1]: Thank you for the feedback. Keep in mind.

**Pull request merged**